lerna.json

```diff
@@ -10,7 +10,7 @@
       "conventionalCommits": true,
       "message": "chore(release): publish",
       "changelogPreset": "conventionalcommits",
-      "gitRemote": "upstream",
+      "gitRemote": "origin",
```

Comment:
question:

Why is this change needed?

Reply:
It seems that the upstream git remote isn't defined in GitHub Actions.